docs/tasks/semester-checklist.rst

```diff
@@ -1,9 +1,39 @@
 Semester Checklist
 ==================
-
 A quick checklist of tasks that need to be completed at the start of every semester.
 
--  Reserve the meeting room by speaking to the lab manager (requires a signature from the president and vice president). Get set up with swipe access if need be.
+Before Semester
+---------------
+Tasks to do during the break.
+
+-  Hold weekly/bi-weekly eboard meetings.
+-  Discuss plans and brainstorm for next semester.
+-  Reserve meeting space.
+-  Triage and review tasks from previous semester to plan for coming semester (see project board).
```

Comment:
Would you mind linking to the project board here too? To do this in reStructuredText, it would look like the following:

```rest
(see `project board <https://github.com/orgs/RITlug/projects/1?fullscreen=true>`__).
```